contracts/compounder/rewardHandlers/HandlerBase.sol

```diff
@@ -17,16 +17,20 @@ contract HandlerBase is IRewardHandler {
     address public immutable token;
     address public immutable strategy;
 
-    address public constant WETH_TOKEN = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    address public immutable WETH_TOKEN;
+    IBalancerVault public immutable balVault;
 
-    address public constant BAL_VAULT = 0xBA12222222228d8Ba445958a75a0704d566BF2C8;
-
-    IBalancerVault public balVault = IBalancerVault(BAL_VAULT);
-
-    constructor(address _token, address _strategy) {
+    constructor(
+        address _token,
+        address _strategy,
+        address _balancerVault,
```

Comment:
Can we make the argument names have the same names as their storage vars. eg `balVault` and `_balVault`